src/fragments/guides/utilities/logger-cloudwatch.mdx

```diff
@@ -0,0 +1,121 @@
+## Using Cloudwatch
+
+You can setup logger to log to Cloudwatch:
+### Extend IAM permissions (without roles)
+
+
+```
+amplify custom project
+```
+Edit the file amplify/backend/awscloudformation/override.ts, and set it to this (be sure to replace your account id):
+```javascript
+import { AmplifyRootStackTemplate } from "@aws-amplify/cli-extensibility-helper";
+
+export function override(resources: AmplifyRootStackTemplate) {
+  const authRole = resources.authRole;
+
+  const basePolicies = Array.isArray(authRole.policies) ? authRole.policies : [authRole.policies];
+  authRole.policies = [
+    ...basePolicies,
+    {
+      policyName: "amplify-permissions-custom-resources",
+      policyDocument: {
+        Version: "2012-10-17",
+        Statement: [
+          {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": "logs:PutLogEvents",
+            "Resource": "arn:aws:logs:*:<ACCOUNT ID>:log-group:weblogs:log-stream:*"
+          },
+          {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+              "logs:CreateLogStream",
+              "logs:DescribeLogGroups",
+              "logs:DescribeLogStreams",
+              "logs:CreateLogGroup"
+            ],
+            "Resource": "arn:aws:logs:*:<ACCOUNT ID>:log-group:weblogs"
```

Comment:
can we create a reference to the account ID here to mitigate the need of hardcoding? perhaps `cdk.Stack.of(this).account`?